tool/README.MD

```diff
@@ -0,0 +1,16 @@
+ ## Generate the json configuration file used by GNI tool
+ 
+ * [util.py][1]: functions used for building the property-value tree
+ * [stat_config.proto][2]: configuration file for generating [stat_config_pb2.py][3], used for parsing the input config files. 
+ * [run_local.sh][4]: commands for running the script locally
+ 
+ * [pop_obs_spec_common.textproto][5]:  input config file
+ * [pop_obs_spec_gni.textproto][6]: input config file
+
+[1]: https://github.com/qlj-lijuan/website/blob/master/tool/util.py
```

Comment:
try to see if there is a way to refer to relative link. Or maybe even remove the link since it's in the same folder